effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -466,20 +468,20 @@ class RecursiveDescent(positions: Positions, tokens: Seq[Token], source: Source)
   def effectDef(): Def =
     nonterminal:
       // effect <NAME> = <EFFECTS>
-      EffectDef(`effect` ~> idDef(), maybeTypeParams(), `=` ~> effects())
+      EffectDef(`effect` ~> idDef(), maybeTypeParams().unspan, `=` ~> effects())
 
   // effect <NAME>[...](...): ...
   def operationDef(): Def =
     nonterminal:
       `effect` ~> operation() match {
         case op @ Operation(id, tps, vps, bps, ret) =>
-          InterfaceDef(IdDef(id.name) withPositionOf op, tps, List(Operation(id, Nil, vps, bps, ret) withPositionOf op))
+          InterfaceDef(IdDef(id.name, id.span) withPositionOf op, tps, List(Operation(id, Many.empty(tps.span.asSynthesized), vps, bps, ret) withPositionOf op))
```

Comment:
Minor nitpick: we often use the pattern `asXYZ` for unsafe casts that panic, which I thought would be happening here. Maybe just call this method `synthesized`?

Reply:
Renamed.